tools/bookrunner/librustdoc/clean/mod.rs

```diff
@@ -1434,188 +1434,7 @@ fn normalize<'tcx>(cx: &mut DocContext<'tcx>, ty: Ty<'_>) -> Option<Ty<'tcx>> {
 impl<'tcx> Clean<Type> for Ty<'tcx> {
     fn clean(&self, cx: &mut DocContext<'_>) -> Type {
         trace!("cleaning type: {:?}", self);
-        let ty = normalize(cx, *self).unwrap_or(*self);
```

Comment:
Yeah, I was looking into this.

One was Rustonomicon Atomics. I haven't investigated yet.

The other was "crate_visibility_modifier" which is an expected new failure.

Reply:
The Atomics one unwinds indefinitely until it times out.

```
while lock.compare_and_swap(false, true, Ordering::Acquire) { }
```

IDK

Reply:
The trigger/regression here seems to be the `Arc`. If I replace it with just a bare `AtomicBool`, it succeeds again.

Reply:
Also of note: It succeeds with `--unwind 0`, so what's changed here is just something that make CBMC not realize it can stop unwinding, not otherwise changing the semantics of the code.

Reply:
Added, and opened https://github.com/model-checking/kani/issues/1276